code/modules/mob/living/simple_animal/friendly/farm_animals.dm

```diff
@@ -80,6 +80,25 @@
 		qdel(GS)
 		eaten = TRUE
 
+	var/mob/living/carbon/human/PP = locate(/mob/living/carbon/human) in oviewers(1, src)
+	var/list/nommable_parts = list()//a list that'll store the limbs of our victim
+	var/time_between_nom = 0 //variable to keep track of the delay between each bite
+	var/nom_delay = 60 //the actual delay
+	if(PP)
+		if(istype(PP.dna.species, /datum/species/pod))//is the species of the mob a podperson?
+			for(var/Q in PP.bodyparts) //getting the victim's current body aprts
+				var/obj/item/bodypart/NN = Q
+				if(NN.body_part != CHEST && NN.body_part != HEAD) //getting every limb but the chest & head
+					nommable_parts += NN //adding the limbs we got to the above-mentioned list
+			if(time_between_nom <= world.time && nommable_parts) //checking if the delay is over & if the victim actually has any parts to nom

```

Comment:
the list always exists, this check does nothing to prevent ```pick()``` runtimes.

Reply:
How exactly would you check to see if a list is empty, then? I assumed that was how it was done.

Reply:
You've check the list exists, just as you would with any other object, which in DM, is ```if(object)``` (```if(object != null)``` in more 'real' languages)

The list will always exist however, since you wrote ```var/list/nommable_parts = list()```, so theres always a valid list. Do see if a list (that exists) is empty, you check that ```list.len``` is not 0, which is as simple as ```if(list.len)```